api/services/ocean/spark/schemas/oceanSparkApplicationSpec.yaml

```diff
@@ -2524,7 +2524,7 @@ components:
           title: Detail
           type: array
           items:
-            "$ref": "#/components/schemas/ValidationError"
```

Comment:
Please explain why you removed this one.

Reply:
I removed this one since it caused the following error:

Semantic error at components.schemas.oceanSparkApplicationSpec.components.schemas.HTTPValidationError.properties.detail.items.$ref
$refs must reference a valid location in the document

However, I did some further investigation and it seems that this location - #/components/schemas/ValidationError - does exist, so the error should not have been caused. Googling showed that it might be a bug in the Swagger Editor:

https://community.smartbear.com/t5/Swagger-Open-Source-Tools/ref-in-response-example-relative-to-response-example/td-p/188875

So I will re-add the location (in a more suitable fashion). Anyway, it seems that this schema is not used anywhere (perhaps was removed in some version, idk) so it is not that crucial.